SCClassLibrary/Common/Control/BelaServerOptions.sc

```diff
@@ -0,0 +1,44 @@
+BelaServerOptions {
+
+	classvar defaultValues;
+
+	*initClass {
+		defaultValues = IdentityDictionary.newFrom(
+			(
+				numAnalogInChannels: 2,
+				numAnalogOutChannels: 2,
+				numDigitalChannels: 16,
+				headphoneLevel: -6,
+				pgaGainLeft: 10,
+				pgaGainRight: 10,
+				speakerMuted: 0,
+				dacLevel: 0,
+				adcLevel: 0,
+				numMultiplexChannels: 0,
+				belaPRU: 1,
+                belaMaxScopeChannels: 0
```

Comment:
make sure to be using all tabs

Reply:
I need a script ASAP for this, I always fuck it up somehow by switching between scide, atom and vim :'(